environment.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Bare minimum setup of the environment to use Vanilla's classes.
+ *
+ * @author Alexandre (DaazKu) Chouinard <alexandre.c@vanillaforums.com>
+ * @copyright 2009-2017 Vanilla Forums Inc.
+ * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
+ */
+
+if (PHP_VERSION_ID < 50600) {
+    die('Vanilla requires PHP 5.6 or greater.');
+}
+
+// Define the constants we need to get going.
+if (!defined('APPLICATION')) {
+    define('APPLICATION', 'Vanilla');
+}
+if (!defined('APPLICATION_VERSION')) {
+    define('APPLICATION_VERSION', '2.4.200');
+}
+if (!defined('DS')) {
+    define('DS', DIRECTORY_SEPARATOR);
+}
+if (!defined('PATH_ROOT')) {
+    define('PATH_ROOT', __DIR__);
```

Comment:
I'm not opposed to switching up `getcwd` for `__DIR__`, but it does break setups/configs that currently work in Vanilla. It might be worth putting a blurb in the release notes for whatever version this gets rolled into.

In tests with PHP 5.6 (I didn't test 7+), even when using $realpath_root in the server config, `getcwd` and `__DIR__` can potentially give different values. If I symlink an index.php from a repo into my web root, `__DIR__` will be the real file dir. `getcwd` will give the web root.